src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineComputer.java

```diff
@@ -108,6 +113,14 @@ protected ComputeEngineCloud getCloud() {
         return node.getCloud();
     }
 
+
+
+    private Operation.Error createSnapshot(ComputeEngineCloud cloud) throws IOException, InterruptedException {
```

Comment:
All this method seems to do is wrap client.createSnapshot(). I'm not sure what value it provides vs in-lining at the call site.